airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/dag.json

```
@@ -0,0 +1,97 @@
+{
+  "allRuns": "所有運行",
+  "blockingDeps": {
+    "dependency": "依賴(Dependencies)",
+    "reason": "原因",
+    "title": "依賴(Dependencies)阻礙任務排程"
+  },
+  "code": {
+    "bundleUrl": "套件包網址",
+    "noCode": "找不到程式碼",
+    "parsedAt": "解析時間："
+  },
+  "extraLinks": "額外連結",
+  "grid": {
+    "buttons": {
+      "resetToLatest": "重設為最新",
+      "toggleGroup": "切換群組狀態"
+    }
+  },
+  "header": {
+    "buttons": {
+      "dagDocs": "Dag 文件"
+    }
+  },
+  "logs": {
+    "noTryNumber": "沒有嘗試次數",
+    "viewInExternal": "在 {{name}} 中檢視日誌(嘗試 {{attempt}})"
+  },
+  "overview": {
+    "buttons": {
+      "failedRun_one": "失敗的執行",
+      "failedRun_other": "失敗的執行",
+      "failedTask_one": "失敗的任務",
+      "failedTask_other": "失敗的任務",
+      "failedTaskInstance_one": "失敗的任務實例",
+      "failedTaskInstance_other": "失敗的任務實例"
+    },
+    "charts": {
+      "assetEvent_one": "已建立資產事件",
+      "assetEvent_other": "已建立資產事件"
```

```suggestion
      "assetEvent_one": "已建立資源事件",
      "assetEvent_other": "已建立資源事件"
```

I think we use 資源 elsewhere

---

Right, I'll fix it real quick

---

This comment seems to be missed